### PR TITLE
refactor(api): remove remaining tip probe related code

### LINKED ISSUE #6805: 4.0.0 Cleanup: Remove tip probe middleware

### PR DESCRIPTION
**Pull request by Laura-Danielle**
# Overview

Closes #6805. 

# Changelog

- Remove tip probe info from robot configs
- Remove and/or deprecate methods specific to tip probe

# Review requests

Anything I missed? Other stuff you would like to see?

# Risk assessment

Low, this feature is no longer accessible to users via the app so the only way this could break someone's protocol is if they were calling things directly.

### CODE REVIEW

File: api/src/opentrons/config/robot_configs.py [hunk 1]
```diff
@@ -217,50 +175,17 @@ def _default_probe_dimensions():
...
-            'switch_offset', [])),
-        switch_clearance=tip_probe_settings.get('switch_clearance',
-                                                TIP_PROBE_SWITCH_CLEARANCE),
-        z_clearance=_build_z_clearance(tip_probe_settings.get('z_clearance',
-                                                              {})),
-        center=tip_probe_settings.get('center', _default_probe_center()),
-        dimensions=tip_probe_settings.get('dimensions',
-                                          _default_probe_dimensions())
-    )
+def build_fallback_instrument_offset(robot_settings: dict) -> dict:
```
Comment (sfoster1) [inline api/src/opentrons/config/robot_configs.py]: this shows up green, weird git thing?

File: api/src/opentrons/hardware_control/api.py [hunk 2]
```diff
@@ -1852,37 +1778,40 @@ async def locate_tip_probe_center(
         The return value is a dict containing the updated position, in deck
         coordinates, of the tip probe center.
         """
+        self._log.info("This function is deprecated."
```
Comment (sfoster1) [inline api/src/opentrons/hardware_control/api.py]: Let's just delete this. Anybody who's using this is using it on purpose and having it not do anything will break their workflow just as much as deleting it

**Review by sfoster1 — CHANGES_REQUESTED** [review]
Couple tiny little changes

File: api/src/opentrons/hardware_control/api.py (hunk 2 above)
Comment (Laura-Danielle) [reply api/src/opentrons/hardware_control/api.py]: okie dokie

File: api/src/opentrons/config/robot_configs.py (hunk 1 above)
Comment (Laura-Danielle) [reply api/src/opentrons/config/robot_configs.py]: No I think it's a weird rebase thing.

**Review by sfoster1 — APPROVED** [review]
lgtm! love it!

**Pull request merged**